WeakAuras/GenericTrigger.lua

```diff
@@ -1131,8 +1131,8 @@ function HandleEvent(frame, event, arg1, arg2, ...)
   end
   if (event == "PLAYER_ENTERING_WORLD") then
     timer:ScheduleTimer(function()
-      Private.StartProfileSystem("generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD");
       HandleEvent(frame, "WA_DELAYED_PLAYER_ENTERING_WORLD", arg1, arg2)
+      Private.StartProfileSystem("generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD");
```

Comment:
this seems unintentional

Reply:
HandleEvent run StartProfileSystem / StopProfileSystem on "generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD" itself, so figured there is no need to do it on there

Reply:
(yes it's slightly offtopic for the description of this commit)